### PR TITLE
chore(slack): migrate slack inbound to generator

### PR DESCRIPTION
**Pull request by igpetrov**
chore(slack): migrate slack inbound to generator

### CODE REVIEW

File: connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java [hunk 1]
```diff
@@ -8,14 +8,39 @@
...
+            group = "endpoint",
+            description =
+                "Slack signing secret. <a href='https://api.slack.com/authentication/verifying-requests-from-slack' target='_blank'>See documentation</a> regarding the Slack signing secret")
+        @NotBlank
+        String slackSigningSecret,
+    @TemplateProperty(
+            id = "verificationExpression",
+            group = "endpoint",
+            type = PropertyType.Hidden,
+            feel = Property.FeelMode.disabled,
```
Comment (chillleader) [inline connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java:38]: Shouldn't it be `required`? Looks like we're expecting a FEEL expression here

File: connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java [hunk 2]
```diff
@@ -8,14 +8,39 @@
...
 public record SlackWebhookProperties(
-    String context,
-    String slackSigningSecret,
-    Function<Map<String, Object>, VerifiableWebhook.WebhookHttpVerificationResult>
-        verificationExpression) {
+    @TemplateProperty(
+            id = "context",
+            label = "Webhook ID",
+            group = "endpoint",
+            description = "The webhook ID is a part of the URL endpoint")
```
Comment (chillleader) [inline connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java:23]: I would also add `feel = FeelMode.disabled` to this property and `slackSigningSecret`. We can change the default behavior for inbound connectors to set `disabled` (As we discussed last week with @Oleksiivanov)

Comment (chillleader) [reply connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java:23]: I created this issue, let's rather change the default value: https://github.com/camunda/connectors/issues/2282

I will tackle it later this week (currently focused on a different task), or feel free to take the lead if you feel like it :)

File: connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java (hunk 1 above)
Comment (igpetrov) [reply connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java:38]: The field is hidden, so it doesn't really matter, and if we remove disabled, it sets `feel: optional` which is a bit confusing. In general, Slack has very specific verification expression, so we sort of preventing user to mess with it.

File: connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java (hunk 2 above)
Comment (igpetrov) [reply connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java:23]: Good point, I'll change that to `disabled`

Comment (igpetrov) [reply connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java]: @chillleader it seems like `disabled` is not working for inbound connectors. Setting `feel = Property.FeelMode.disabled` had no effect. Except the field is `hidden`, then it works. 🤔

Comment (chillleader) [reply connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java]: > Setting feel = Property.FeelMode.disabled had no effect.

Good catch, I could confirm the issue - here is the proposed fix: https://github.com/camunda/connectors/pull/2286

**Pull request merged**